Makefile

```diff
@@ -14,10 +14,11 @@ clean:
 # Process .ego templates (skippable)
 EGOS := $(shell find . -name *.ego | sed -e s/.ego/.ego.go/)
 $(EGOS):
+	go get github.com/benbjohnson/ego/cmd/ego
 	ego api/views
 
 # Fetch dependencies
-vendor: glide.yaml
+init: $(EGOS)
```

Comment:
i'm unsure about this change. would you make the `init` rule separate? my thinking was that `make vendor` should only run if `glide.yaml` has been recently modified. (https://www.gnu.org/software/make/manual/make.html#Prerequisite-Types)

Reply:
I don't think that logic applies here. If it was back the way it was `vendor: glide.yaml` and run `make vendor` then simply run `make vendor` again you will see that it still runs `glide install` and so on, even though `glide.yaml` is unchanged.  

What you are actually doing is making that target dependent on `glide.yaml` as a prerequisite. So if `glide.yaml` is not present you get this error: `make: *** No rule to make target 'glide.yaml', needed by 'vendor'.  Stop.`

I was unclear of that until now so I will adding back

Reply:
My understanding is that `vendor: glide.yaml` will run when a) the target is missing or b) the prerequisite is newer. Here's how it looks in my environment:

```bash
$ make vendor
make: `vendor' is up to date.
$ touch glide.yaml
$ make vendor
glide install
...
go install
$ make vendor
make: `vendor' is up to date.
```

Reply:
what version on Make?

Mine:
```
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```

Reply:
huh:

```
$ make -v
GNU Make 3.81
Copyright (C) 2006  Free Software Foundation, Inc.
This is free software; see the source for copying conditions.
There is NO warranty; not even for MERCHANTABILITY or FITNESS FOR A
PARTICULAR PURPOSE.

This program built for i386-apple-darwin11.3.0
```